flows/codelingo/review/review/review.go

```diff
@@ -41,12 +33,14 @@ func RequestReview(ctx context.Context, req *flow.ReviewRequest, insecure bool)
 	}
 
 	util.Logger.Debug("sending request to flow server...")
-	replyc, runErrc, err := c.Run(ctx, &flow.Request{Flow: "review", Payload: payload})
+	replyc, runErrc, cancel, err := flowutil.RunFlow("review", payload, func() proto.Message { return &flow.Reply{} })
 	if err != nil {
 		return nil, nil, errors.Trace(err)
 	}
 	util.Logger.Debug("...request to flow server sent. Received reply channel.")
 
+	_ = cancel
```

Comment:
debug code? 

If we're not using cancel, assign the returned value directly to "_" on line 36